wesnoth-did.po

```diff
@@ -4313,24 +4231,22 @@ msgid ""
 "themselves to orcs than follow my ‘sorcerous ways’. Drogan, Dela... all of "
 "them! They are the ones without souls, not me!"
 msgstr ""
-"Pcha! Tuhle hru můžou hrát dva. Vždycky jsem bojoval, jen abych chránil svůj "
-"lid a svou rodinu, ale za to jsem si vysloužil jen opovržení a vyhnanství. "
-"Vlastní sestra mě nebude poslouchat. Moji lidé raději nakrmí skřety sami "
-"sebou, než aby chápali mé ‚čarodějnické způsoby‘. Drogan, Dela... ti "
-"všichni! To oni jsou bez duše, ne já!"
+"Pcha! Nezasloužím si to a určitě neodpovídám na řeči někoho jako je ona. "
+"Vždycky jsem bojoval, jen abych chránil svůj lid a svou rodinu, ale za to "
```

Comment:
tak jak to je, mi tam ta čárka přebývá, ale nevím proč - vizte třeba jen aby v [téhle povídce](https://www.vsehrd.cz/clanek/vzpominky-na-davneho-pritele_19f3f96e-2788-4791-9499-468d429a1b78)

jo kdyby tam bylo bojoval jen proto, abych... tak tam je jasná
```suggestion
"Vždycky jsem bojoval jen abych chránil svůj lid a svou rodinu, ale za to "
```